echopype/echodata/echodata.py

```diff
@@ -188,14 +188,27 @@ def __get_dataset(self, node: DataTree) -> Optional[xr.Dataset]:
             return node.ds
         return None
 
+    def __get_node(self, key: Optional[str]) -> DataTree:
+        if key in ["Top-level", "/"]:
+            # Access to root
+            return self._tree
+        return self._tree[key]
+
     def __getitem__(self, __key: Optional[str]) -> Optional[xr.Dataset]:
         if self._tree:
             try:
-                if __key in ["Top-level", "/"]:
-                    # Access to root
-                    node = self._tree
-                else:
-                    node = self._tree[__key]
+                node = self.__get_node(self, __key)
```

Comment:
```suggestion
                node = self.__get_node(__key)
```

I think `self ` should be removed? Unless this is like the absolute import situation?